packages/kolibri-common/components/SearchChips.vue

```diff
@@ -33,16 +33,23 @@
   import flatMap from 'lodash/flatMap';
   import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
   import { NoCategories } from 'kolibri.coreVue.vuex.constants';
-  import useChannels from '../composables/useChannels';
-  import useLanguages from '../composables/useLanguages';
+  import useChannels from 'kolibri-common/composables/useChannels';
+  import { injectBaseSearch } from 'kolibri-common/composables/useBaseSearch';
 
   export default {
     name: 'SearchChips',
     mixins: [commonCoreStrings],
     setup() {
-      const { languagesMap } = useLanguages();
+      const { availableLanguages } = injectBaseSearch();
+      const languagesMap = availableLanguages.reduce((map, lang) => {
```

Comment:
Ah yes, this is confirmed by @radinamatic's testing - this needs to either `get` the `availableLanguages` or access the `.value`.